weaviate_cli/commands/create.py

```diff
@@ -292,6 +299,7 @@ def create_data_cli(
     randomize,
     auto_tenants,
     vector_dimensions,
+    uuid,
 ):
     """Ingest data into a collection in Weaviate."""
 
```

Comment:
Add a check here please that if limit > 1 and uuid is not none, throw an error. Or if you don't want to restrict it with an error, override the limit value to 1 if uuid is not none.

Reply:
Added, also a check that randomized is paased